docs/authentication.rst

```diff
@@ -442,18 +442,17 @@ any action. We may implement specialized tokens in the future.
 Token Security Considerations
 `````````````````````````````
 
-This section is for information only. Token length and encoding may be subject
-to change in the future.
+This section is for information only. Token length and encoding may change in
+the future.
 
-Any token is generated from 168 bits of true randomness at the server. Guessing
-the token correctly is hence practically impossible. The value corresponds to 21
-bytes and is represented by 28 characters in Base64-like encoding. That is, any token
-will only consist of URL-safe characters ``A-Z``, ``a-z``, ``-``, and ``.``. (We do not
-have any padding at the end because the string length is a multiple of 4.)
+Any token is generated from 168 bits of randomness at the server and stored in
+hashed format (PBKDF2-HMAC-SHA256). Guessing the token correctly or reversing
+the hash is hence practically impossible.
 
-As all tokens are stored in plain text on the server, the user may not choose
-the token value individually to prevent re-using passwords as tokens at deSEC.
+The token value is represented by 28 characters using a URL-safe variant of
+base64 encoding. It comprises only the characters ``A-Z``, ``a-z``, ``-``, and
+``.``. (Base64 padding is not needed as the string length is a multiple of 4.)
```

Comment:
again, this should probably cherry-pick 6460c4e867844702dfdd111237b70ad37c695cb4

Reply:
planning to rebase on top of other branch